content/applications/sales/sales/send_quotations/orders_and_variants.rst

```diff
@@ -1,78 +1,113 @@
-===================================================
-Use product variants on quotations and sales orders
-===================================================
+===============================================
+Product variants on quotations and sales orders
+===============================================
 
-Product variants can be added to quotations and sales orders using the Product Configurator,
-Variant Grid Entry, or both.
+Product variants can be added to quotations and sales orders using the *Product Configurator*,
+*Variant Grid Entry*, or both.
 
 Configuration
 =============
 
-To enable product variants on quotations and sales orders, go to :menuselection:`Sales -->
-Configuration --> Settings --> Product Catalog` and activate one or both of the
-:guilabel:`Product Configurator` and :guilabel:`Variant Grid Entry` settings. Finally, click
-:guilabel:`Save` to apply the settings.
+To enable product variants on quotations and sales orders, go to :menuselection:`Sales app -->
+Configuration --> Settings`, and under the :guilabel:`Product Catalog` heading, activate one (or
+both) of the :guilabel:`Product Configurator` and :guilabel:`Variant Grid Entry` settings. Finally,
+click :guilabel:`Save` to apply the settings.
 
 .. image:: orders_and_variants/activating-entry-type-settings.png
    :align: center
    :alt: Activating entry type settings.
 
-After saving the settings, configure the product form. Head over to :menuselection:`Sales -->
-Products --> Products` and select any product (or click :guilabel:`Create` to create a new one).
-On the product form, click :guilabel:`Edit`, and then click on the :guilabel:`Variants` tab. Under
-:guilabel:`Sales Variant Selection`, choose either :guilabel:`Product Configurator` or
+After saving the settings, it's time to configure the product form.
+
+To do that, navigate to :menuselection:`Sales --> Products --> Products`, and select the desired
```

Comment:
In the previous menu selection tab, you used "Sales app" instead of just "Sales". Either way is fine, but it's probably good to make it consistent throughout the doc.

Reply:
great catch!